v2.0/src/config/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```

Comment:
we shouldnt be checking code in as is local

Reply:
Right - but this is still in local test - not to be merged